mozilla-release/browser/components/AutoPrivateTab.js

```diff
@@ -335,7 +335,13 @@ AutoPrivateTabDatabase.prototype = {
     tab.private = true;
     tabBrowser.reload();
 
-    if (remember) {
+    if (domain) {
```

Comment:
Fur internal pages with URLs like "about:*" or "chrome:*" the domain is undefined, so we shouldn't even attemp to cleanup anything based on it, hence no user prompt is needed as well.